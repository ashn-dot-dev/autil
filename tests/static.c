#define AUTIL_IMPLEMENTATION
#define AUTIL_API static

#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wunused-function"

#include "../autil.h"

int
main(void)
{
    puts("AUTIL COMPILED WITH STATIC LINKAGE");
    return EXIT_SUCCESS;
}
