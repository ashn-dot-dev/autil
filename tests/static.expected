AUTIL COMPILED WITH STATIC LINKAGE
